frontend/src/multiAdd/MultiAdd.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import AddIcon from "@material-ui/icons/Add";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import MultiAddItem from "./MultiAddItem";
+import FilledInput from "@material-ui/core/FilledInput";
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      width: "195px",
+      margin: theme.spacing(1),
+    },
+    addIcon: {
+      cursor: "pointer",
+    },
+  })
+);
+
+interface IMultiAddProps {
+  title: string;
+  placeholder: string;
+  callback: (items: string[]) => void;
+}
+
+function MultiAdd(props: IMultiAddProps) {
```

Comment:
in the future, we can also spread props as `function MultiAdd({title, placeholder, callback} as IMultiAddProps)` then use the variables directly. not necessary